crates/replace/src/lib.rs

```diff
@@ -326,19 +320,12 @@ impl<T: Config> Pallet<T> {
 
         let requestable_tokens = ext::vault_registry::requestable_to_be_replaced_tokens::<T>(&vault_id)?;
         let to_be_replaced_increase = amount_btc.min(&requestable_tokens)?;
-        let replace_collateral_increase = if amount_btc.is_zero() {
-            griefing_collateral
-        } else {
-            ext::vault_registry::calculate_collateral::<T>(&griefing_collateral, &to_be_replaced_increase, &amount_btc)?
-        };
+
+        ensure!(!amount_btc.is_zero(), Error::<T>::ReplaceAmountZero);
```

Comment:
```suggestion
        ensure!(!to_be_replaced_increase.is_zero(), Error::<T>::ReplaceAmountZero);
```
Can you also add a test for this? Fyi, I think adding zero btc was allowed because it allowed you to increase the griefing collateral, but with your change it no longer makes sense